steps/household/rent.tsx

```diff
@@ -150,6 +166,28 @@ const step: ProcessStepDefinition<ProcessDatabaseSchema, "household"> = {
           },
         },
       },
+      {
+        label: questions["payment-card"],
+        values: {
+          "payment-card": {
+            renderValue(hasAppliedforPaymentCard: string): React.ReactNode {
+              return getRadioLabelFromValue(
+                rentPaymentCardRadios,
+                hasAppliedforPaymentCard
+              );
+            },
+          },
+          "payment-card-notes": {
+            renderValue(notes: Notes): React.ReactNode {
+              if (notes.length === 0) {
+                return;
+              }
+
+              return <ReviewNotes notes={notes} />;
+            },
+          },
+        },
+      },
```

Comment:
@LBHABEBIC are these changes supposed to be here? are they from that blocked ticket that you have?

Reply:
No they are not meant to be here. darn it.